### ISSUE TITLE
Wrong extrusion suggested as spacer on page 17

### ISSUE THREAD

**Issue by rafal-molotkiewicz** [VoronDesign/Voron-Trident#146]
Page 17 calls to use Extrusion E as a spacer. Extrusion E is horizontal and spec-dependent. It is OK on 250 spec, as the E has 330mm, but on 300 and 350mm spec E is 380 or 430mm long, respectively. Information that it only works for 250 spec should be more prominent, at least.

With a little rearrangement of the assembly order, I was able to use Extrusion H as a spacer, something that's faster, cheaper and more friendly to the environment than printing something that will be used only once and then discarded, Please consider it as an option.

**Comment by nemgrea** [comment]
Duplicate of #77
